Source/Fuse.Controls.Panels/Backgrounds/TopFrameBackground.uno

```diff
@@ -49,7 +49,13 @@ namespace Fuse.Controls
 
 		protected override float2 GetContentSize(LayoutParams lp)
 		{
-			if defined(iOS || Android)
+			if defined(iOS) {
+				// on iOS, we always treat the bar as though it's 40 points
+				var height = 40 / AppBase.Current.PixelsPerPoint;
```

Comment:
It just seems very odd that a PR which is supposed to adjust to a dynamic change in the status bar is using a static size for it?!  This also seems not to be future proof, as I suspect Apple isn't guaranteeing a permanent 40pixel height to the status bar.